pkg/apis/v1beta1/api.go

```diff
@@ -109,5 +111,9 @@ func (a *APISpec) Validate() []error {
 		errors = append(errors, fmt.Errorf("%s is not a valid address for sans", a))
 	}
 
+	if !govalidator.IsIP(a.Address) {
+		errors = append(errors, fmt.Errorf("%v is not IP address", a.Address))
```

Comment:
Let the user also know which field it was that failed. Now they only see that some address was not in correct form but have (maybe) no idea which field it was.